components/DashboardPage.tsx

```diff
@@ -19,8 +19,28 @@ export default function DashboardPage() {
 	const [whitelist, setWhitelist] = useState<string[]>([]);
 	const [editedWhitelist, setEditedWhitelist] = useState<string[]>([]);
 	const [whitelistHtml, setWhitelistHtml] = useState<JSX.Element[] | JSX.Element>([]);
+	const [minimize, setMinimize] = useState({});
```

Comment:
```suggestion
	const [minimize, setMinimize] = useState<any>({});
```
This will set it as `any` for everything after this. Or you can do it fancy like this:
```suggestion
	const [minimize, setMinimize] = useState<{ [key: string]: boolean }>({});
```

Reply:
Implemented